language/move-prover/spec-lang/src/translate.rs

```diff
@@ -613,6 +616,28 @@ impl<'env> Translator<'env> {
                     result_type: param_t.clone(),
                 },
             );
+            add_builtin(
+                self,
+                self.builtin_fun_symbol("borrow_global"),
```

Comment:
We can go with this for now, but what is a problem is that users can now also call these functions in their spec funs. They shouldn't be able to do so, because this leads to confusion. Better would be we would be mapping calls to those to global. But as said, for not postponing the PR, we can do it. Please add a TODO.